### PR TITLE
PTD-599 Modify BTC icon in Binance and RSK Icon

### PR DESCRIPTION
**Pull request by cryptoiner**
- [x] Updated BTCB icon 
- [x] Updated RSK icon

### CODE REVIEW

**Review by djaciel — APPROVED** [review]
LGTM!

File: src/attributes/tokens.ts [hunk 1]
```diff
@@ -108,7 +108,7 @@ export const TokensAttributes: { [tokenSymbol: string]: ITokenAttributes } = {
   },
   WBTC: {
     symbol: 'WBTC',
-    logoURL: 'https://assets.coingecko.com/coins/images/7598/small/wrapped_bitcoin_wbtc.png',
+    logoURL: 'https://bscscan.com/token/images/btcb_32.png',
```
Comment (pmprete) [inline src/attributes/tokens.ts:111]: The correct WBTC icon is the one from coingeko 
```suggestion
    logoURL: 'https://assets.coingecko.com/coins/images/7598/small/wrapped_bitcoin_wbtc.png',
```

**Review by pmprete — APPROVED** [review]
Please correct the WBTC icon, except that everything is ok

File: src/attributes/tokens.ts (hunk 1 above)
Comment (cryptoiner) [reply src/attributes/tokens.ts]: Done!

**Pull request merged**